src/main.cc

```diff
@@ -195,8 +194,7 @@ static int daemon_main(LiteClient& client, const bpo::variables_map& variables_m
         std::string reason = "Updating from " + current.filename() + " to " + found_latest_target->filename();
```

Comment:
Not sure github is including the context for my comment: We have this line:
~~~
if (!known_target_sha && !client.isTargetActive(*found_latest_target)) {
~~~

Its calling `getCurrent` under the covers. I wonder if we should drop that and roughly do
~~~
if (!known_target_sha && current != found_latest_target) {
~~~